core/state_transition.go

```diff
@@ -193,6 +193,7 @@ func (st *StateTransition) buyGas() error {
 	if st.gasFeeCap != nil {
 		balanceCheck = new(big.Int).SetUint64(st.msg.Gas())
 		balanceCheck = balanceCheck.Mul(balanceCheck, st.gasFeeCap)
+		balanceCheck.Add(balanceCheck, st.value)
```

Comment:
Just FYI this change makes this comment stale https://github.com/ethereum/go-ethereum/blob/f68a68a313a7a5806b0e414965e500fef453306a/core/state_transition.go#L267

Reply:
huh? lost